### PR TITLE
Fidelis - minor bug fixes, commands and readme files

### PR DESCRIPTION
**Pull request by roysagi**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23173

## Description
Added Readme file and commands file
Also fixed some minor issues where errors where raised when getting partial results back

### CODE REVIEW

File: Integrations/FidelisElevateNetwork/README.md [hunk 1]
```diff
@@ -0,0 +1,1283 @@
+Automate Detection and Response to Network Threats and data leakage in your organization with Fidelis Elevate Network Integration.
+This integration was integrated and tested with version xx of Fidelis Elevate Network
```
Comment (glicht) [inline Integrations/FidelisElevateNetwork/README.md:2]: Please replace xx with the version we used for testing

Comment (roysagi) [reply Integrations/FidelisElevateNetwork/README.md]: done

**Comment by roysagi** [comment]
@ronykoz -Circle fails here because of a bug in the SDK that does not process the <br> html tag properly - it is a known issue that will be solved in an upcoming SDK version

File: Integrations/FidelisElevateNetwork/README.md [hunk 2]
```diff
@@ -0,0 +1,1283 @@
...
+    }
+}
+```
+
+##### Human Readable Output
+### Alert 9
+|Attribute Map|Clickable Dpaths|Decoding Paths|Original Attributes|Original D Path|
+|---|---|---|---|---|
+| HTTP: {u'endIndex': 29, u'name': u'Filename', u'valueFirst255': u'file.pcap', u'highLights': [], u'value': u'file.pcap', u'startIndex': 0, u'link': False, u'partialAttr': u'HTTP\x0cFilename\tfile.pcap\n'} | HTTP(file.pcap) | {u'clickable': True, u'highLights': [], u'linkPath': u':HTTP(file.pcap)', u'path': u'HTTP(file.pcap)'} | HTTP
Filename	file.pcap<br> | :HTTP(file.pcap) |
```
Comment (yaakovi) [inline Integrations/FidelisElevateNetwork/README.md:1022]:
```suggestion
| HTTP: {u'endIndex': 29, u'name': u'Filename', u'valueFirst255': u'file.pcap', u'highLights': [], u'value': u'file.pcap', u'startIndex': 0, u'link': False, u'partialAttr': u'HTTP\x0cFilename\tfile.pcap\n'} | HTTP(file.pcap) | {u'clickable': True, u'highLights': [], u'linkPath': u':HTTP(file.pcap)', u'path': u'HTTP(file.pcap)'} | HTTPFilename	file.pcap<br/> | :HTTP(file.pcap) |
```

Comment (yaakovi) [reply Integrations/FidelisElevateNetwork/README.md:1022]: @roysagi this should fix it

**Comment by anara123** [comment]
@kirbles19 please review

**Comment by kirbles19** [comment]
@roysagi Done.

**Pull request merged**